apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js

```diff
@@ -80,6 +87,100 @@ describe('ExportPaymentsService', () => {
+        await createTestMultiPayment(admin, [invoicePayment], admin.user, acquiringIntegration)
+
+        const {
+            data: {
+                result: {
+                    status,
+                    linkToFile,
+                },
+            },
+        } = await admin.query(EXPORT_PAYMENTS_TO_EXCEL, {
+            data: {
+                dv: 1,
+                sender: { dv: 1, fingerprint: 'test-' + faker.random.alphaNumeric(8) },
+                where: { invoice: { organization: { id: organization.id } } },
+                sortBy: 'advancedAt_DESC',
+                timeZone: DEFAULT_ORGANIZATION_TIMEZONE,
+            },
+        })
+        const formatDate = (date) => dayjs(date).tz(DEFAULT_ORGANIZATION_TIMEZONE).format('DD.MM.YYYY HH:mm')
+
+        const filename = getTmpFile('xlsx')
+        await downloadFile(linkToFile, filename)
+        const data = await readXlsx(filename)
+
+        expectDataFormat(data, [
+            ['Дата', 'ЛС', 'Адрес', 'Помещение', 'Тип', 'Транзакция', 'П/П', 'Статус', 'Сумма'],
+            [
+                formatDate(invoicePayment.advancedAt),
+                invoicePayment.accountNumber,
+                '-',
+                '-',
+                acquiringIntegration.name,
+                '',
+                '',
+                i18n('payment.status.' + invoicePayment.status, { locale }),
+                Number(invoicePayment.amount).toFixed(2).toString(),
+            ],
+        ])
+        expect(status).toBe('ok')
+        expect(linkToFile).not.toHaveLength(0)
+
+        const {
+            data: {
+                result: {
+                    status: status2,
+                    linkToFile: linkToFile2,
+                },
+            },
+        } = await admin.query(EXPORT_PAYMENTS_TO_EXCEL, {
+            data: {
+                dv: 1,
+                sender: { dv: 1, fingerprint: 'test-' + faker.random.alphaNumeric(8) },
+                where: {
+                    organization: { id: organization.id },
+                    invoice_is_null: true,
+                },
+                sortBy: 'advancedAt_DESC',
+                timeZone: DEFAULT_ORGANIZATION_TIMEZONE,
+            },
+        })
+        const filename2 = getTmpFile('xlsx')
+        await downloadFile(linkToFile2, filename2)
+        const data2 = await readXlsx(filename2)
+
+        expectDataFormat(data2, [
+            ['Дата', 'ЛС', 'Адрес', 'Помещение', 'Тип', 'Транзакция', 'П/П', 'Статус', 'Сумма'],
+            [
+                formatDate(receiptPayment.advancedAt),
+                receiptPayment.accountNumber,
+                receiptPayment.receipt.property.address,
+                receiptPayment.receipt.account.unitName,
+                acquiringIntegration.name,
+                '',
+                '',
+                i18n('payment.status.' + receiptPayment.status, { locale }),
+                Number(receiptPayment.amount).toFixed(2).toString(),
```

Comment:
Shouldnt we use bigjs here?

Reply:
I think no. I repeated the behavior that is implemented in the export service

Reply:
But you are getting amount which is bigJS field i think, number conversions in JS are tricky, why not use something will work correctly 100% time?

I fear that this test will fail randomly from time to time